src/store/modules/order/actions.ts

```diff
@@ -0,0 +1,49 @@
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import OrderState from './orderState'
+import * as types from './mutation-types'
+import { OrderService } from '@/services/OrderService'
+import { hasError, showToast } from '@/utils'
+import { translate } from '@/i18n'
+
+const actions: ActionTree<OrderState, RootState> = {
+  async getDraftOrder ({ commit }, params) {
+    const payload = {
+      draftOrderId: params.id,
+      shopifyConfigId: params.configId
+    }
+    try {
+      const resp = await OrderService.getDraftOrder(payload);
+      if (resp.status === 200 && !hasError(resp) && resp.data.response.draft_order) {
```

Comment:
If the resp has error then the function will not go in the if block, so we need to have an else block to display the error in console.